### ISSUE TITLE
clean gives a 404 - suspect jobId with "/" character

### ISSUE THREAD

**Issue by remy** [felixmosh/bull-board#307]
I've got job names that include a slash character, such as `user/234` and `server/2` and when I try to clean tasks individually, it fails with a 404.

I suspect it's due to the slash character, but not sure where to start to confirm. Would this be a matter of escaping the jobId before using in the URL?

Example `PUT` request (note that my UI is on `/q`):

```
http://localhost:5555/q/api/queues/core-data/user/234/clean
```

**Comment by felixmosh** [comment]
It can happen :]

**Comment by felixmosh** [comment]
fix Released in `v3.3.3`
